File: 168. Excel Sheet Column Title/solution.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        ans = ""
        while columnNumber > 0:
            temp = columnNumber % 26
            if 0 == temp:
                temp = 26
            ans = chr(64+temp)+ans
            columnNumber = (columnNumber - temp)//26
        return ans

File: 168. Excel Sheet Column Title/solution.php
<?php
class Solution {

    /**
     * @param Integer $columnNumber
     * @return String
     */
    function convertToTitle($columnNumber) {
        if ($columnNumber < 27) {
            return chr($columnNumber + 64);
        }

        $result = '';
        while ($columnNumber > 0) {
            $codeNum = ($columnNumber - 1) % 26;
            $result = chr($codeNum + 65) . $result;
            $columnNumber = intdiv($columnNumber - 1, 26);
        }

        return $result;
    }
}
